.idea/typescript-compiler.xml

```diff
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="TypeScriptCompiler">
     <option name="isCompilerEnabled" value="true" />
-    <option name="useConfig" value="true" />
+    <option name="typeScriptCompilerParams" value="-p tsconfig.json -m system" />
```

Comment:
The compiler needs to be configured to use the tsconfig.json file. If something needs to be configured, configure it there.